cmd.go

```diff
@@ -259,11 +259,23 @@ type Info struct {
 	// For example, if this value is 'option', the default message 'value for flag'
 	// will become 'value for option'.
 	FlagKnownAs string
+
+	// ShowSuperFlags contains the names of the 'super' command flags
+	// that are desired to be shown in the sub-command help output.
+	ShowSuperFlags []string
```

Comment:
I guess we never have any cases where we have one command that can work under multiple supercommands, do we?

Reply:
No, we do not :)